src/operators/concat-static.ts

```diff
@@ -15,7 +15,6 @@ export default function concat<R>(...observables: Array<Observable<any> | Schedu
   let args = <any[]>observables;
   if (typeof (args[observables.length - 1]).schedule === 'function') {
     scheduler = args.pop();
-    args.push(1, scheduler);
```

Comment:
Very weird line of code here, pushing the number `1` into `args` array and pushing `scheduler` back right after having popped it out from `args`.